library/src/scripts/dataLists/dataList.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import React from "react";
+
+import { dataListClasses } from "@library/dataLists/dataListStyles";
+import { t } from "@vanilla/i18n/src";
+import classNames from "classnames";
+
+export interface IData {
+    key: React.ReactNode;
+    value: React.ReactNode;
+}
+
+export interface IDataList {
+    className?: string;
+    data: IData[];
+}
```

Comment:
Do we need to export this? Doesn't seem to be used anywhere so maybe we can just do the standard `IProps`